pages/resources/channel.mdx

```diff
@@ -45,6 +45,8 @@ Represents a guild or private channel within Discord.
 | default_reaction_emoji?             | ?[default reaction](#default-reaction-object) object               | The emoji to show in the add reaction button on a thread in a thread-only channel                                                                                                            |
 | default_forum_layout?               | integer                                                            | The default [layout](#forum-layout-type) of a thread-only channel                                                                                                                            |
 | default_sort_order?                 | ?integer                                                           | The default [sort order](#sort-order-type) of a thread-only channel's threads (default `LATEST_ACTIVITY`)                                                                                    |
+| icon_emoji?                         | ?[icon emoji](#icon-emoji-object) object                           | The emoji to show next to the channel name in channels list                                                                                                                                  |
+| theme_color?                        | ?integer                                                           | The background color of the channel icon emoji encoded as an integer representation of hexadecimal color code                                                                                |
```

Comment:
```suggestion
| theme_color?                        | ?integer                                                           | The background color of the channel icon emoji encoded as an integer hex color code                                                                                |
```

Reply:
This is the way it's written everywhere else in the docs. I think it should be kept for consistency.